.github/workflows/daily.yml

```diff
@@ -16,23 +16,25 @@ jobs:
       fail-fast: false
       matrix:
         #this is a painful solution since it doesn't pick up new branches, other option is lotsa shell in one job....
-        target-branch: [master, release-1.1]
+        #to test in canary add in canary here.....
+        target-branch: [master, release-1.1, canary]
```

Comment:
Change to use wildcard for the release branch so that we can set it up once?   

For context, we would normally keep 3 branches: master, release, pre-release.  We want alerts in all of them.

Reply:
wont work, unfortunately.   "target-branch" is the name of a variable.